src/Sylius/Bundle/CoreBundle/Doctrine/ORM/ProductReviewRepository.php

```diff
@@ -0,0 +1,39 @@
+<?php
+
+/*
+ * This file is part of the Sylius package.
+ *
+ * (c) Paweł Jędrzejewski
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Sylius\Bundle\CoreBundle\Doctrine\ORM;
+
+use Sylius\Bundle\ResourceBundle\Doctrine\ORM\EntityRepository;
+use Sylius\Component\Review\Model\ReviewInterface;
+
+/**
+ * @author Grzegorz Sadowski <grzegorz.sadowski@lakion.com>
+ */
+class ProductReviewRepository extends EntityRepository
+{
+    /**
+     * @param int $productId
+     *
+     * @return ReviewInterface[]
+     */
+    public function findAcceptedBySubject($productId)
```

Comment:
This method should be in `ReviewRepository` since it's not coupled to `Product`. Rename to `findAcceptedBySubjectId($subjectId)`.